Makefile

```
@@ -39,9 +39,17 @@ verify: #verify-shellcheck ## Runs verification scripts to ensure correct execut
 verify-shellcheck: ## Runs shellcheck
 	./hack/verify-shellcheck.sh
 
-##@ Helpers
-
-.PHONY: help
-
-help:  ## Display this help
-	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+.PHONY: test
+test: test-go test-shellscripts ## Runs unit tests to ensure correct execution
```

```suggestion
test: test-go test-sh ## Runs unit tests to ensure correct execution
```

This is an actual issue, `test-shellscripts` vs `test-sh` ... I don't care which one we use, but I care that we just use one ;)